custom_actions.go

```diff
@@ -246,7 +246,18 @@ func makeCustomActionCall(identifier *string, arguments *[]actionArgument) (cust
 			case String:
 				argumentValue = fmt.Sprintf("\"%s\"", argumentValue)
 			case Variable:
-				argumentValue = fmt.Sprintf("\"{%s}\"", argumentValue)
+				var variableValue, found = getVariableValue(argumentValue)
+				if !found {
+					// Not sure what to do here
+					exit("Variable not found!")
```

Comment:
Because we are in a parsing loop, we would use `parserError()` here. We're using functions and variables in `parser.go` that this function will use to print out where the issue is.

This should also be rephrased to use `fmt.Sprintf()` to include the name of the not found variable if you were not already planning to please.

Reply:
Thank you, should be ready now!